lib/ember-orbit/schema.js

```diff
@@ -151,13 +151,13 @@ var Schema = Ember.Object.extend({
           var rel = record.__rel[link] = {};
           linkValue.forEach(function(id) {
             if (typeof id === 'object') {
-              id = id.primaryId;
+              id = id.get('primaryId');
             }
             rel[id] = true;
           });
 
         } else if (typeof linkValue === 'object') {
-          record.__rel[link] = linkValue.primaryId;
+          record.__rel[link] = linkValue.get('primaryId');
```

Comment:
It is marginally faster (saves one method call), so it's not a bad idea for library code. Also, it works with POJOs or Ember.Objects, so it's slightly more flexible. Neither is particularly important here, but I'm trying to follow conventions throughout ember-orbit that are used within ember itself.